http/src/main/resources/reference.conf

```diff
@@ -154,7 +154,7 @@ vpc {
     }
   ]
   # Remove RDP and SSH rules in the default network. Also remove legacy leonardo-notebooks-rule if it exists.
-  firewallsToRemove = ["default-allow-rdp"]
+  firewallsToRemove = ["default-allow-rdp", "default-allow-icmp", "allow-icmp"]
```

Comment:
`default-allow-icmp` exists in "regular" projects; `allow-icmp` exists in high-security AoU projects.

Will update https://github.com/broadinstitute/gcp-dm-templates so these fw rules aren't created in the first place for new projects.